.github/workflows/pre-merge.yaml

```diff
@@ -53,3 +53,8 @@ jobs:
 
       - name: Build detekt
         run: ./gradlew build
+
+
+      - name: Run smoke test
+        run: ./gradlew testPluginKotlinc
+        if: ${{ runner.os != 'Windows' }}
```

Comment:
Runs the test added in #6 

As a TODO it should be fixed on Windows, the problem isn't with this plugin, but the format of the CLI commands - quoting certain things is required on Windows CLI that aren't needed on Linux or macOS.